disorder-fsm/src/Disorder/FSM.hs

```diff
@@ -1,9 +1,14 @@
 {-# LANGUAGE NoImplicitPrelude #-}
 {-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```

Comment:
What's the current opinion on `GeneralizedNewtypeDeriving`? I though it was considered to be relatively unsafe.